app/graphql/types/input/account/set_site_permissions.rb

```diff
@@ -0,0 +1,14 @@
+class Types::Input::Account::SetSitePermissions < Types::Input::Base
+  argument :account_id, ID,
+    required: true,
+    description: 'Who to grant permissions to',
+    as: :account
+
+  argument :permissions, [Types::Enum::SitePermission],
+    required: true,
+    description: 'The permissions to set for this user'
+
+  def load_account(value)
+    ::User.find(value)
+  end
```

Comment:
Do we want this here, or inside the mutation?

Reply:
Here, because we use it in `authorized?`

Reply:
Ooooh, I see what you've done. If you want to go that route shouldn't we rename the `value` argument to `id` to be more specific?

Also unless I am thinking of this wrong from the `load_account` it takes what that returns and passes that along to the `authorized?` method (which would be a user) not an input? 

I'll test this locally tomorrow because either this won't work at all or you are about to teach me something new (and I'm hoping it is the latter because this really opens up the possibilities).

Reply:
I don't think I'm doing anything weird, I'm pretty sure I've just copied it from your stuff… my thinking is that this should run on the input type to load the record and then when `authorized?` runs it's available as input.account

Reply:
So the `load_*` isn't actually related to an `input`. Its the graphql-ruby gem magic and it must be in the mutation file itself.